components/Subscribe/IFTTTItem.js

```diff
@@ -34,88 +38,196 @@ const useStyles = makeStyles(theme => ({
       top: '0',
     },
   },
-  textField: { padding: '0 14px' },
+  textFieldInput: {
+    fontSize: theme.typography.caption.fontSize,
+  },
+  subscribe: {
+    fontSize: theme.typography.h6.fontSize,
+    [theme.breakpoints.up('md')]: {
+      fontSize: theme.typography.h5.fontSize,
+    },
+    borderRadius: 30,
+    padding: '3px 30px',
+    textAlign: 'center',
+    border: 'none',
+  },
 }));
 
 const SUCCESS = 'SUCCESS';
 
+function TabPanel(props) {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`simple-tabpanel-${index}`}
+      aria-labelledby={`simple-tab-${index}`}
+      {...other}
+    >
+      {value === index && <Box>{children}</Box>}
+    </div>
+  );
+}
+
+TabPanel.propTypes = {
+  children: PropTypes.node,
+  index: PropTypes.any.isRequired,
+  value: PropTypes.any.isRequired,
+};
+
+function a11yProps(index) {
+  return {
+    id: `simple-tab-${index}`,
+    'aria-controls': `simple-tabpanel-${index}`,
+  };
+}
+
 const DialogBody = forwardRef(function DialogBody(props, ref) {
   const { IFTTTAppletUrl, feedUrl, tutorialYoutubeId, platform, close } = props;
   const classes = useStyles();
   const [message, setMessage] = useState('');
   const [status, setStatus] = useState(null);
+  const [value, setValue] = useState(0);
+
+  const CustomTab = withStyles(theme => ({
```

Comment:
We can move the definition of `CustomTab` and `CustomCopyButton` to outside of `DialogBody`'s render function